### PR TITLE
Home

### PR DESCRIPTION
**Pull request by a-sh-dev**
- created `<Header >` component in `component/layout` that can be used in other pages for consistency
- modified posts and post to work around routing
- current individual postcard when click will be linked to about page for testing only →  this routing is still on progress to be configured

### CODE REVIEW

File: src/components/styles/DIYtheme.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ const theme = (mode) =>
...
         fontFamily: global.fontSecondary,
       },
+      headerTitle: {
+        fontFamily: global.fontPrimary,
+        fontStyle: 'italic',
+        fontSize: '1.5rem',
+        letterSpacing: -0.5,
+      },
     },
   });
```
Comment (NatachaBartu) [inline src/components/styles/DIYtheme.js:116]: nice implementation around global styling.

File: src/views/home/Home.js [hunk 2]
```diff
@@ -22,13 +25,27 @@ function Home() {
...
-      <h1>Home Page</h1>
-      <h3>Searchbar</h3>
-      <h3>Featured Posts</h3>
-      <hr />
-      <Posts />
-    </Container>
+    <>
+      <Box
+        component="header"
+        sx={{
```
Comment (NatachaBartu) [inline src/views/home/Home.js]: Are we inline style ? or we should use style.js file?

**Comment by thangzathang** [comment]
Looks great 👍. Changed a bit of the card structure?

**Comment by a-sh-dev** [comment]
Thank you team. I added a `<CardActionArea>` I think, so that the card can be clicked to link it to the post. Styling still need to be finaslied later

**Pull request merged**